scripts/deployment/debian/control

```diff
@@ -10,6 +10,4 @@ Package: nethermind
 Architecture: amd64
 Multi-Arch: foreign
 Depends: libsnappy-dev, libc6-dev, libc6, unzip, jq, curl, wget
-Description: .NET Core Ethereum client full and actively developed
```

Comment:
This description is made up of two parts. The first one is a synopsis/short description and second part is a longer description.